intellst-api/src/Services/IdentifiedCaseHandler.php

```diff
@@ -81,4 +102,92 @@ public function updateIdentifiedCaseAllowEntrance(AllowEntranceDTO $dto, Identif
 
         return $errors;
     }
+
+    public function getIdentifiedCase(): array
+    {
+        $user = $this->userHandler->getList();
+        $temperature = $this->returnUserEnterprise($user)->temperature;
+        $data = $this->returnUserEnterprise($user)->restrictionPeriod;
```

Comment:
`$data` isn't a good name. If it is a number of days, then `$days` if it is DateTime, then `$date`